index.bs

```diff
@@ -110,6 +110,11 @@ spec: page-visibility; urlPrefix: https://www.w3.org/TR/page-visibility/
 spec: WHATWG HTML; urlPrefix: https://html.spec.whatwg.org/
     type: dfn 
         text: focus
+
+spec: FIDO-APPID; urlPrefix: https://fidoalliance.org/specs/fido-u2f-v1.2-ps-20170411/fido-appid-and-facets-v1.2-ps-20170411.html
```

Comment:
shall we leverage https://fidoalliance.org/specs/fido-v2.0-rd-20170927/fido-client-to-authenticator-protocol-v2.0-rd-20170927.html instead? it is what's ref'd in the biblio.

Reply:
That's CTAP2, which doesn't define an AppID nor FacetID. However, I do see that there's a `FIDO-APPID` reference already in the biblio and I'm not sure what is the right thing to do here. Based on other cases, I've deleted the current biblio entry because it'll get populated automatically based on this chunk.

Reply:
I corrected my mistake in the prior comment. apologies.